src/test/Makefile

```diff
@@ -481,17 +481,12 @@ GTEST_DIR = ../../lib/test/gtest
 
 # Use clang/clang++ by default
 
-CC  := clang-12
-CXX := clang++-12
-ifneq ($(OSFAMILY), linux)
-ifeq ($(shell which $(CC) 2>/dev/null),)
-$(info Falling back to 'clang' on Windows and OSX.)
-CC  := clang
-CXX := clang++
+ifeq ($(origin CC), default)
+CC  = clang
 endif
```

Comment:
This change would make clang default to version 14 on Ubuntu without the default set to use `clang-15`

Reply:
See also https://github.com/betaflight/betaflight/pull/13507#discussion_r1557887869